baselines/Google Chat Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -300,3 +300,72 @@ Agencies SHOULD configure DLP rules to block or warn on sharing files with sensi
 9.  Select the appropriate action to warn or block sharing, based upon the agency's individual requirements.
 10. In the **Alerting** section, choose a severity level, and optionally, check **Send to alert center to trigger notifications**.
 11. Review the rule details, mark the rule as **Active**, and click **Create.**
+
+## 7. Content Reporting
+
+This section covers the content reporting functionality.
+
+### Policies
+
+#### GWS.CHAT.7.1v0.1
+Chat Content Reporting SHALL be Enabled.
```

Comment:
Change spelling, feature is not capitalized like this in Google's documentation, e.g.,
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/15df9aa0-b128-4705-99b2-cef997ef4ad2)


```suggestion
Chat content reporting SHALL be enabled.
```